build.gradle

```diff
@@ -1,6 +1,6 @@
 buildscript {
 	dependencies {
-		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "latest.release"
+		classpath group: "com.liferay", name: "com.liferay.gradle.plugins.defaults", version: "6.1.77"
```

Comment:
We should remove this commit. This should be fixed by https://github.com/brianchandotcom/liferay-portal/pull/80545.